lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -83,18 +83,103 @@ describe('NxCombobox', function() {
         expect(inputElement).toHaveValue('Foo');
       });
 
-  it('sets aria-expanded on the input to true when focused',
+  it('sets aria-expanded to true on the input when initially focused', async function() {
```

Comment:
Just like the predetermined list vs backend query cases in the functional test, this `aria-expanded` behavior is only true when the `matches` is populated, right? We should have another test for when it isn't

Reply:
> > Just like the predetermined list vs backend query cases in the functional test, this aria-expanded behavior is only true when the matches is populated, right? We should have another test for when it isn't
> 
> Should probably also have cases that check `aria-expanded` when the loading, empty, and error alerts are supposed to be present

Added in extra testing here: [1d7f7bf](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/1d7f7bfa6d42416ba7ba9ba6eebb44ad66b462d0)